fill/fill_station.cc

```diff
@@ -72,6 +70,15 @@ class CommanderServiceImpl final : public Commander::Service
         if (request->qd_retract()) {
              qd.Actuate();
         }
+        if (request->sv2_close()){
```

Comment:
This seems out of place in this commit

Reply:
Yeah that's my bad, I meant to do it in the PR above. I can fold the PRs if you think that is necessary

Reply:
Folding doesn't make much sense here I don't think. I'd recommend just removing it here and keeping it in the PR above

Reply:
Ok, then if I delete this branch that will still contain the changes from the PR above?

Reply:
When you delete these changes on this branch, it will ask you to restack the one above it. When you are addressing the merge conflicts, you can make sure the changes stay in that commit.